packages/infra/lib/api-stack.ts

```diff
@@ -1080,6 +1089,41 @@ export class APIStack extends Stack {
     return documentDownloaderLambda;
   }
 
+  /**
+   * We are intentionally not setting an alarm action for this lambda, as many issues
+   * may be caused outside of our system. To eliminate noise, we will not alarm on this
+   * lambda.
+   */
+  private setupBulkUrlSigningLambda(ownProps: {
+    lambdaLayers: LambdaLayers;
+    vpc: ec2.IVpc;
+    bucketName: string | undefined;
+    envType: EnvType;
+    sentryDsn: string | undefined;
+  }): Lambda {
+    const { lambdaLayers, vpc, bucketName, sentryDsn, envType } = ownProps;
+
+    const bulkUrlSigningLambda = createLambda({
+      stack: this,
+      name: "BulkUrlSigning",
+      runtime: lambda.Runtime.NODEJS_18_X,
+      entry: "bulk-url-signing",
+      envType,
+      envVars: {
+        ...(bucketName && {
+          MEDICAL_DOCUMENTS_BUCKET_NAME: bucketName,
+        }),
+        ...(sentryDsn ? { SENTRY_DSN: sentryDsn } : {}),
+      },
+      layers: [lambdaLayers.shared],
+      memory: 512,
```

Comment:
unsure if is sufficient memory

Reply:
I think we might want to increase to 1GB, some patients have a lot of records.